documents/ack/ack-cube-game.yml

```diff
@@ -0,0 +1,450 @@
+ROSTemplateFormatVersion: '2015-09-01'
+Description:
+  en: Deploy cube game using ACK.
+  zh-cn: 使用ACK部署魔方游戏。
+Parameters:
+  AckName:
+    Type: String
+    Label:
+      en: Cluster Name
+      zh-cn: 集群名称
+    Description:
+      en: The name must be 1 to 63 characters in length and can contain letters, Chinese
+        characters, digits, and hyphens (-).
+      zh-cn: 名称为1~63个字符，可包含数字、汉字、英文字符或中划线（-）。
+    Default: ack-demo
+    AllowedPattern: ^[a-zA-Z0-9\u4e00-\u9fa5][-a-zA-Z0-9\u4e00-\u9fa5]{0,62}$
+  ZoneId:
+    Type: String
+    Label:
+      en: VSwitch Available Zone
+      zh-cn: 可用区
+    AssociationProperty: ALIYUN::VPC::Zone::ZoneId
+  InstanceType:
+    Type: String
+    Label:
+      en: Instance Type
+      zh-cn: 节点规格
+    AssociationProperty: ALIYUN::ECS::Instance::InstanceType
+    AssociationPropertyMetadata:
+      ZoneId: ${ZoneId}
+    AllowedPattern: ^(ecs\.[a-z5-9]+\.)[1-9]?xlarge$
+  SystemDiskCategory:
+    Type: String
+    Label:
+      en: Worker System Disk Category
+      zh-cn: 节点系统盘磁盘类型
+  LoginPassword:
+    Type: String
+    Label:
+      en: Instance Password
+      zh-cn: 节点登录密码
+    Description:
+      en: |-
+        The password must be 8 to 32 characters in length. <br>
+        It must consist three of the the following character types: uppercase letters, lowercase letters, digits, and special characters. <br>
+        Special characters include <span style="background:#E7E9EB;"><b>()`~!@#$%^&*_-+=|{}[]:;'<>,.?/</b></span>.<br>
+      zh-cn: 长度为8-30位，需包含大写字母、小写字母、特殊符号和数字中的三个，允许的特殊字符包括<span style="background:#E7E9EB;"><b>()`~!@#$%^&*_-+=|{}[]:;'<>,.?/</b></span>。
+    AssociationProperty: ALIYUN::ECS::Instance::Password
+Resources:
+  EnableAck:
+    Type: ALIYUN::ROS::AutoEnableService
+    Properties:
+      ServiceName: CS
+  AliyunCSManagedLogRole:
+    Type: ALIYUN::RAM::Role
+    Properties:
+      RoleName: AliyunCSManagedLogRole
+      Description: 容器服务(CS) Kubernetes 集群日志组件使用此角色来访问您在其他云产品中的资源
```

Comment:
Please add `。` and check similiar Description.

Reply:
Copied by existed role description，there is no `。`  :smile: